.devcontainer/postCreateCommand.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+export DEBIAN_FRONTEND=noninteractive
+sudo apt-get -y --force-yes update
+sudo apt-get -y --force-yes install libgdal-dev
+echo CPLUS_INCLUDE_PATH=/usr/include/gdal >> ~/.bashrc
+echo C_INCLUDE_PATH=/usr/include/gdal >> ~/.bashrc
```

Comment:
Wheels for rasterio/fiona/etc. should come with GDAL, did you encounter issues with installing TorchGeo's dependencies?

Reply:
Yes, on my side gdal was needed to use TorchGeo in my container

Reply:
What was the error message when GDAL was not installed?

Reply:
> What was the error message when GDAL was not installed?

Failed to get options via gdal-config: 
WARNING:root:Failed to get options via gdal-config: [Errno 2] No such file or directory: 'gdal-config'

When installing fiona